care/emr/resources/questionnaire/utils.py

```
@@ -153,39 +153,56 @@ def is_question_enabled(question, responses, questionnaire_obj):  # noqa PLR0912
             or not responses[condition_question_id].values
         ):
-            condition_value = None
+            condition_values = []
         else:
-            condition_value = responses[condition_question_id].values[0].value
+            # Get all values for the question
+            values = responses[condition_question_id].values
+            # For choice questions, check all selected values
+            condition_values = [v.value for v in values]
 
         condition_value = normalize_boolean_value(condition_value)
```

Ruff (0.11.9)

Local variable `condition_value` is assigned to but never used

Remove assignment to unused variable `condition_value`

(F841)